samples/bluetooth/hci_uart_async/src/hci_uart_async.c

```diff
@@ -82,14 +74,14 @@ static int uart_c2h_tx(const uint8_t *data, size_t size)
 }
 
 /* Function expects that type is validated and only CMD, ISO or ACL will be used. */
-static uint32_t hci_payload_size(const uint8_t *hdr_buf, enum h4_type type)
+static uint32_t hci_payload_size(const uint8_t *hdr_buf, uint8_t type)
```

Comment:
```suggestion
static uint32_t hci_payload_size(const uint8_t *hdr_buf, uint8_t h4_type)
```

The type lost the 'h4' hint. I think we should move it to the parameter name to retain self-documentation.

Reply:
> The type lost the 'h4' hint. I think we should move it to the parameter name to retain self-documentation.

Sure. I'll ad it back.

Reply:
> > The type lost the 'h4' hint. I think we should move it to the parameter name to retain self-documentation.
> 
> Sure. I'll add it back.

Done.